code/game/objects/items/weapons/toggle.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/weapon/toggle
+	var/active = FALSE
+	var/force_on = 0
+	var/throwforce_on = 0
+	var/icon_state_on = "null"
+	var/list/attack_verb_on = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	var/w_class_on = 4
+	var/armour_penetration_on = 0
+	var/activation_sound
+	var/active_hitsound
+
+/obj/item/weapon/toggle/attack_self(mob/living/carbon/user)
+	if(active)
+		active = FALSE
+		force = initial(force)
+		throwforce = initial(throwforce)
+		hitsound = initial(hitsound)
+		attack_verb = list()
+		icon_state = initial(icon_state)
+		item_state = initial(item_state)
+		armour_penetration = initial(armour_penetration)
+		w_class = initial(w_class)
```

Comment:
Varediting any of the 'off' vars won't stick around after toggling and I think that's dumb but it's not really your fault and fixing it would take a bunch more vars